teachertool/src/transforms/runEvaluateAsync.ts

```diff
@@ -94,11 +104,21 @@ export async function runEvaluateAsync(fromUserInteraction: boolean) {
                     return resolve(false);
                 }
 
-                const planResult = await runValidatorPlanAsync(plan, loadedValidatorPlans);
+                // Only call into iframe if teacher tool has not specified an override for this plan.
+                let planResult = await runValidatorPlanOverrideAsync(plan);
+                if (!planResult) {
+                    planResult = await runValidatorPlanAsync(plan, loadedValidatorPlans);
+                }
 
                 if (planResult) {
-                    const result = planResult.result ? EvaluationStatus.Pass : EvaluationStatus.Fail;
-                    setEvalResultOutcome(criteriaInstance.instanceId, result);
+                    const result =
+                        planResult.result === undefined
```

Comment:
Does this need to be a nested ternary? This will always boil down to the result being pass or fail, right? Can the update here then be planResult.result === undefined?

Reply:
It does not always boil down to pass or fail. If planResult.result is undefined, then result is CompleteWithNoResult. If planResult.result is set, then we decide pass/fail.

Reply:
Oh, gotcha. Thanks!